dp.html

```diff
@@ -57,7 +57,7 @@
     <div class="container">
         <div class="text-center">
             <h2 class="section-heading text-uppercase">Nous contacter</h2>
-            <h3 class="section-subheading text-muted">Une question ? Une proposition ? Juste en-dessous ça passe.</h3>
+            <h3 class="section-subheading text-muted">Une question ? ça se passe juste ici.</h3>
```

Comment:
I've asked you to write "Cela se passe juste en-dessous." and not "ça se passe juste ici.".